os/arch/arm/src/s5j/s5j_dma.h

```diff
@@ -78,61 +78,61 @@
 
 /* CCR BITFIELDS */
 
-#define NOSWAP                  (0)
-#define ENDIAN_SWAP_SIZE_MASK   (7 << 28)
-#define ENDIAN_SWAP_SIZE(x)     ((x & 7) << 28)
+#define NOSWAP 			(0)
```

Comment:
align with below.

Reply:
We had better rollback to previous version.
This file also modified by ahreum before.

Reply:
done